modules/services/workload-scanning/variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "external_id" {
+  description = "Random string generated unique to a customer"
+  type        = string
+}
+
+variable "trusted_identity" {
+  type        = string
+  description = "The name of sysdig trusted identity"
+}
+
+variable "name" {
+  description = "The name of the installation. Assigned to most child resource(s)"
+  type        = string
+  default     = "sysdig-workload-scanning"
+}
+
+variable "tags" {
+  type        = map(string)
+  description = "sysdig secure-for-cloud tags. always include 'product' default tag for resource-group proper functioning"
+  default = {
+    "product" = "sysdig-secure-for-cloud"
+  }
+}
+
+variable "deploy_global_resources" {
```

Comment:
suggestion: I have added conditional validation to my PR to make sure the dependent values are provided when `deploy_global_resources` is set to true. Might be worth considering adding something similar here as well.

https://github.com/draios/terraform-aws-secure-for-cloud/pull/52/files#diff-eb8c58cc3cf81193ad0246ca55c00a824de2764a893091e9764353a5d27d8a0cR52